en_US/messages.json

```
@@ -138,5 +138,21 @@
     "tlsIncompatibility": {
       "message": "Firefox Translations only works on secure (HTTPS) pages.",
       "description":"Informs the user that the extension doesn't work on non-https pages."
+    },
+    "datacollectionConsentPageTitle": {
+      "message": "Allow Firefox Translations to send technical and interaction data to Mozilla?",
+      "description": "Header of the data collection consent page"
+    },
+    "datacollectionConsentPageDescription": {
+      "message": "This includes high-level user interaction and error reports. Changes to this setting can be made any time in preferences. Data is stored securely and without personally identifying information.",
```

```suggestion
      "message": "This includes high-level user interaction and error reports. Changes to this setting can be made any time in settings. Data is stored securely and without personally identifying information.",
```

We moved from options/preferences to settings, I think it's worth using that here.